docs/api/graph_ops.rst

```diff
@@ -0,0 +1,31 @@
+.. py:module:: vortex.runtime.onnx.graph_ops
+.. py:currentmodule:: vortex.runtime.onnx.graph_ops
+
+ONNX graph ops utility
+======================
+
+Collections of utilty to manipulate onnx graph.
+
+.. autoclass:: vortex.runtime.onnx.graph_ops.embed_metadata.EmbedMetadata
+  :members:
+
+  .. automethod:: __init__
```

Comment:
![image](https://user-images.githubusercontent.com/18413229/113539606-57b02b00-9608-11eb-8986-d60a432d4e80.png)

I dont think you need to add all the init, because it is already been shown in the title, it's just a bit redundant

Reply:
ok

Reply:
updated